packages/client/src/hooks/useNFTs.js

```diff
@@ -100,6 +139,16 @@ export default function useNFTs() {
     }).catch(console.error);
 
     const collections = (identifiers && identifiers[1]) ?? [];
+
+    if (collections.length === 0) {
+      dispatch({
+        type: "SET_NFTS",
+        payload: {
+          [treasuryAddr]: {},
+        },
+      });
+    }
+
     for (const identifier in collections) {
       await checkCollection(treasuryAddr, collections[identifier]);
```

Comment:
Since each checkCollection will fire-and-forget async, having await here won't do anything, suggest remove